apps/fhir/bluebutton/views/home.py

```diff
@@ -121,45 +116,7 @@ def get_supported_resources(resources, resource_names, resource_router=None):
         for k, v in item.items():
             if k == 'type':
                 if v in resource_names:
-                    filtered_item = get_interactions(v, item, resource_router)
-                    # logger.debug("Filtered Item:%s" % filtered_item)
-
-                    resource_list.append(filtered_item)
+                    item['interaction'] = [{"code": "read"}, {"code": "search-type"}]
```

Comment:
This makes `resource_router` unused, so it should be removed.

Reply:
Good point. Fixed.